Tools/CISupport/ews-build/events.py

```diff
@@ -164,13 +222,46 @@ def buildFinished(self, key, build):
             "steps": build.get('steps'),
         }
 
-        self.sendData(data)
+        self.sendDataToEWS(data)
+
+    @defer.inlineCallbacks
+    def stepStartedGitHub(self, build, state_string):
+        sha = self.extractProperty(build, 'revision')
+        repository = self.extractProperty(build, 'repository')
+        if not sha or not repository:
+            return
+
+        builder = yield self.master.db.builders.getBuilder(build.get('builderid'))
+
+        self.sendDataToGitHub(
+            repository, sha, dict(
```

Comment:
Ditto, might consider making the dictionary in a separate statement (before self.sendDataToGitHub invocation) for better readability.